week3/Week3HomeWork/Week3HomeWork.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Are you intentionally using the net5.0 version here? 😊

Reply:
Yes, I have done it with my work computer which has a net5.0 version. :)